# Obstacle Extractor
# To -> map
obstacle_detector_to_map: 
  ros__parameters:
    active: true
    use_scan: true
    use_pcl: false

    use_split_and_merge: true
    circles_from_visibles: true
    discard_converted_segments: false
    transform_coordinates: true

    min_group_points: 50

    max_group_distance: 0.04
    distance_proportion: 0.00628
    max_split_distance: 0.02
    max_merge_separation: 0.25
    max_merge_spread: 0.02
    max_circle_radius: 0.2
    radius_enlargement: 0.05

rival_localization:
  ros__parameters:
    robot_name: robot
    rival_name: rival
    frequency: 100.0
    x_max: 2.9
    x_min: 0.1
    y_max: 1.9
    y_min: 0.1   
    vel_lpf_gain: 0.9
    locking_rad: 0.5
    lockrad_growing_rate: 0.5 # m/s
    